proofs/eo/cpc/rules/Quantifiers.eo

```diff
@@ -379,3 +379,73 @@
   :requires ((($is_quant_dt_split x ($dt_get_constructors (eo::typeof x)) ys F G) true))
   :conclusion (= (forall (@list x ys) F) G)
 )
+
+;;;;; ProofRule::EXISTS_INV_CONDITION
+
+; program: $mk_inv_cond_op_disj
+; args:
+; - f (-> (BitVec n) (BitVec n)): The unary function to apply, e.g. (bvashr s).
+; - t (BitVec n): The right hand side of the equation.
+; - w Int: The upper bound of bit-vector constants to consider.
+; - n Int: The current constant we are considering.
+; return: >
+;    A disjunct used in the invertibility conditions for some shift operators.
+(program $mk_inv_cond_op_disj ((n Int) (f (BitVec n)) (n Int) (t (BitVec n)) (w Int))
+  ((-> (BitVec n) (BitVec n)) (BitVec n) Int Int) (BitVec n)
+  (
+  (($mk_inv_cond_op_disj f t w w)  (eo::cons or (= (f (eo::to_bin w w)) t) false))
+  (($mk_inv_cond_op_disj f t w n)  (eo::cons or (= (f (eo::to_bin w n)) t) ($mk_inv_cond_op_disj f t w (eo::add n 1))))
+  )
+)
+
+; program: $mk_invertibility_condition
+; args:
+; - x (BitVec n): A bitvector variable to solve for.
+; - R Bool: A relation involving x.
+; return: >
```

Comment:
```suggestion
; return: 
```

Reply:
Actually as I understand it, this is the formatting required for multi-line fields in the yaml (although we haven't leveraged this formatting yet).